src/vs/workbench/parts/terminal/electron-browser/terminal.contribution.ts

```diff
@@ -322,6 +324,14 @@ actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(HideTerminalFind
 	primary: KeyCode.Escape,
 	secondary: [KeyCode.Shift | KeyCode.Escape]
 }, ContextKeyExpr.and(KEYBINDING_CONTEXT_TERMINAL_FOCUS, KEYBINDING_CONTEXT_TERMINAL_FIND_WIDGET_VISIBLE)), 'Terminal: Focus Find Widget', category);
+actionRegistry.registerWorkbenchAction(new SyncActionDescriptor(NextMatchTerminalFindWidgetAction, NextMatchTerminalFindWidgetAction.ID, NextMatchTerminalFindWidgetAction.LABEL, {
+	primary: KeyCode.F3
+}, KEYBINDING_CONTEXT_TERMINAL_FOCUS), 'Terminal: Find Next', category);
```

Comment:
Usually for macOS, we use another keybinding -> https://github.com/Microsoft/vscode/blob/master/src/vs/editor/contrib/find/common/findController.ts#L372